app/controllers/healthcare_controller.rb

```diff
@@ -1,6 +1,10 @@
 class HealthcareController < AssessmentsController
+  before_action :skip_intro_if_police, only: %i[intro]
```

Comment:
Let's not introduce a before_action for that. This can be just changing the not_started_path value of the options to pass to render the partial in ```app/views/escorts/show.html.slim```
```
    = render 'assessment', assessment: escort.risk,
                           model: Risk,
                           not_started_path: (escort.from_prison? ? new_escort_risks_path(escort) : intro_escort_risks_path(escort)),
                           summary_path: escort_risks_path(escort),
                           section: escort.risk || escort.build_risk
```